src/main/python/prunetags/cli.py

```diff
@@ -92,6 +92,13 @@ def get_parser():
         dest='execute',
         action='store_true',
         help='execute processing - not setting is same as running in NOOP mode')
+    parser.add_argument(
+        '--version',
```

Comment:
the value of the `version` argument would be very nice to include in the mpcurses screen

Reply:
it would be nice to have the CLI validate version prior to invoking the API, that way if the user enters an invalid expression for the version the CLI would catch it immediately as opposed to having the API fail.

Reply:
Added to mpcurses and added corresponding tests.

I change the verbiage a bit if the --version option is used to not include 'pre-release'
![image](https://user-images.githubusercontent.com/61600932/89591665-a00c1480-d7ff-11ea-887c-e351dc324886.png)